src/marqo/tensor_search/tensor_search.py

```diff
@@ -1022,6 +1112,10 @@ def _vector_text_search(
         except (KeyError, IndexError) as e2:
             raise e
 
+    logger.info(f"  search (tensor) Marqo-os processing time: took {(total_os_process_time):.3f}s for Marqo-os to execute all searches.")
+
```

Comment:
Can we say 

`took ... for Marqo-os to execute the search`

Marqo-os does a parallel `_msearch`, but this is an implementation detail we can abstract away from the Marqo user.

Reply:
done.